lib/test_bot.rb

```diff
@@ -52,6 +52,7 @@ def run!(args)
         raise UsageError, "cannot use --cleanup from HOMEBREW_PREFIX as it will delete all output."
       end
 
+      ENV["HOMEBREW_BOOTSNAP"] = "1"
```

Comment:
Just do double-check, is this no-op on ARM now too? I know it is when you're on portable-ruby, but I wasn't sure about ARM.

Reply:
Yes. Homebrew/brew makes it a no-op anywhere it doesn't work.